commands/utility/get_user_info.js

```diff
@@ -0,0 +1,54 @@
+const { SlashCommandBuilder, PermissionFlagsBits } = require('discord.js');
+/*  ----------------------
+Google Sheets Setup
+-------------------------- */
+const { spreadsheetId } = require('../../config.json');
+const { google } = require('googleapis');
+
+const auth = new google.auth.GoogleAuth({
```

Comment:
Does this require auth on every call for get_user_info command? or is it just executed once (mb idk how it works)

Reply:
not totally sure -- will look into it n fix if needed